custom_components/versatile_thermostat/thermostat_switch.py

```diff
@@ -121,6 +123,7 @@ async def async_added_to_hass(self):
                     self.hass, [switch.entity_id], self._async_switch_changed
                 )
             )
+            switch.startup()
```

Comment:
ok. Thank for the explanation.

> “Please note that keep-alive messages will only start after the thermostat is first switched on,”

So, if I understand well, in summer, when Vtherm is off, the keep-alive will continue to be send to the device ? Am I wrong ? Why not but I wonder how to avoid that. I consider permanently running features even if turned off is a bad pattern. But maybe I understand a wrong way.

Reply:
> “VTherm” is short for “Versatile Thermostat”, right?

Yes